_sass/module/_candidate.scss

```diff
@@ -1,9 +1,23 @@
 $candidate-photo-width: 10rem;
 
-.candidate__occupation {
+.candidate__occupation__commitee_name {
   @include heading('h3');
   color: $subheading-color;
   font-style: italic;
+  font-size: 18px;
+  margin-bottom: 5px;
```

Comment:
Start with [`$spacing-base`](https://github.com/caciviclab/odca-jekyll/blob/3a4a6e034a1a23f57a2863b97a6585b647634515/_sass/base/_elements.scss#L1) and then a `0.5` multiplier if necessary instead of specifying absolute sizes here.